internal/controllers/module_nmc_reconciler.go

```diff
@@ -402,6 +408,37 @@ func (mnrh *moduleNMCReconcilerHelper) removeModuleFromNMC(ctx context.Context,
 	return nil
 }
 
+func (mnrh *moduleNMCReconcilerHelper) moduleUpdateWorkerPodsStatus(ctx context.Context, mod *kmmv1beta1.Module, targetedNodes []v1.Node) error {
```

Comment:
Do you think this function could use the NMC labels `module-configured` and `module-in-use`? Would it make things simpler maybe?

Reply:
those labels won't help since they don't signify if the kernel module has been already loaded or not. We could try using ready labels on the nodes, but i thought it is more reliable to be dependent on NMC